### DISCUSSION TITLE
[Runtime Filter] Why does the in_list (and min_max) runtime filter apply only to prune partitions

### DISCUSSION THREAD

**Discussion by JasonLi-cn** [databendlabs/databend#14231] [Q&A] [answered]
Why can't in_list runtime filter be applied to row granularity filtering, like bloom runtime filter.🤔

**Comment by bohutang** [comment]
I would like to ping @xudong963 here , he is the Databend runtime filter impler.

**Comment by xudong963** [comment, marked as answer]
`in_list` runtime filter will generate when build side row number is less than 1024, but bloom runtime filter doesn't have the limitation. They have similar effect for row granularity filtering, so using bloom runtime filter is enough.

**Reply by JasonLi-cn** [reply]
BTW, When will the distributed runtime filter be implemented?

**Reply by xudong963** [reply]
Broadcast join has implemented runtime filter, the next step is to support rf for shuffle join
